pkg/image/controller/signature/container_image_downloader.go

```diff
@@ -46,7 +46,8 @@ func (s *containerImageSignatureDownloader) DownloadImageSignatures(image *image
 
 	signatures, err := source.GetSignatures(ctx)
 	if err != nil {
-		return nil, err
+		glog.V(4).Infof("Failed to get signatures for %v due to: %v", source.Reference(), err)
```

Comment:
Hrm, can we make this an ignorable error instead?  I don’t want to make it too complex, but would prefer to eat the error as late as possible.

Another option is a rate limiter on this type of error (if rate exceeded, don’t log).  Maybe burst 100 rate 1/10s or something.

Reply:
other than whatever handleerror does, do we have a utility for managing rate limits of logging/errors?

Reply:
we have a rate limiter class - it's fairly easy to add.  Can be omitted here.